__tests__/api/app.test.js

```diff
@@ -12,35 +13,55 @@ beforeEach(() => {
   return seed(seedTestData);
 });
 
-describe('Testing app', () => {
-  describe('Endpoint /api/topics', () => {
-    describe('Method GET', () => {
-      test('GET: 200 status', () => {
-        return request(app).get('/api/topics').expect(200);
+describe("Testing app", () => {
+  describe("Endpoint /api/topics", () => {
+    describe("Method GET", () => {
+      test("GET: 200 status", () => {
+        return request(app).get("/api/topics").expect(200);
       });
 
-      test('GET: 200 status with all topics data and properties', () => {
+      test("GET: 200 status with all topics data and properties", () => {
         return request(app)
-          .get('/api/topics')
+          .get("/api/topics")
           .expect(200)
```

Comment:
this `.expect(200)` is repeated from the previous test, you can save a line by removing it from this second test